src/main/java/com/internhub/data/positions/pages/PageScraper.java

```diff
@@ -98,7 +89,7 @@ private Position tryGetPosition(Page page, Company company) {
                 childLink = ScraperUtils.fixLink(childLink, page.getLink());
 
                 // Make sure that the link can be transposed to a valid URL
```

Comment:
replace `transposed` with converted or transformed. There's no need to add a more complex word. You can keep it if you want but it's definitely weird.